CMakeLists.txt

```diff
@@ -46,6 +47,9 @@ FIND_PACKAGE(CMLIB
 
 IF (NOT BRINGAUTO_SYSTEM_DEP)
     INCLUDE(cmake/Dependencies.cmake)
+    IF (BRINGAUTO_GET_PACKAGES)
```

Comment:
Can we name variable differently? Something similar to "BRINGAUTO_GET_PACKAGES_ONLY"?

Reply:
variable renamed